images/gams/Dockerfile

```diff
@@ -10,9 +10,6 @@ RUN apt-get install -y git
 
 # copy gams files
 COPY ./gams-install-dir/gamsFiles/docker-install-java.sh /opt/gams/gamsFiles/docker-install-java.sh
-COPY ./gams-install-dir/gamsFiles/test-file.gms /opt/gams/gamsFiles/test-file.gms
```

Comment:
tento test file by som nechal - na nom sa potom testuje pri builde licencia a ci je gams spravne nainstalovany

Reply:
Je v tom bordel - praveze sa to netestuje na tomto suboru. Aj gams-integration aj gams-lambda si kopiruju svoj gms subor. Tento je tam cely cas nepouzity.

Reply:
ano, ale to odtial zmizne takze to treba zjednotit